src/common/controllers/ads-controller.js

```diff
@@ -205,7 +207,9 @@ class AdsController extends FakeEventTarget implements IAdsController {
       this._handlePrebidAdConfig();
       this._handleConfiguredPreroll();
       this._eventManager.listenOnce(this._player, Html5EventType.DURATION_CHANGE, () => {
-        this._handleEveryAndPercentage();
+        this._player.isLive()
+          ? this._eventManager.listenOnce(this._player, Html5EventType.SEEKING, () => this._handleLiveEvery())
```

Comment:
this is the initial seek to live edge once loading? if so in start over mode I'm not we will get it (although I'm  sure start over its live actually)